frontend/scripts/react-components/shared/steps-tracker/steps-tracker.scss

```diff
@@ -112,13 +111,13 @@ $circle-dot-size: $circle-size / 2;
 
     .steps-tracker-item {
       position: relative;
-      display: block;
       width: $circle-size;
       padding: $label-height 0 0;
       height: $item-height;
+      overflow: visible;
```

Comment:
We need to be specific that the content inside will be visible, IE is not clever enough to assume this